### DISCUSSION TITLE
Made a translation into Ukrainian

### DISCUSSION THREAD

**Discussion by soulUA** [edrlab/thorium-reader#1947] [Show and tell]
Hello
I made a translation into Ukrainian
where should i download the file - ua.json ?

**Comment by danielweck** [comment]
Hello, thank you for offering to contribute this language.
The technical language code is `uk-UA` I believe, so my recommendation would be for your to create `uk.json` and place it inside 
the `locales` folder (do this in your GitHub fork of Thorium, from the `develop` branch, and then propose a "Pull Request" here in GitHub ):
https://github.com/edrlab/thorium-reader/tree/develop/src/resources/locales
Can you please also show us how to write the term "ukrainian" using your local script? Thank you :)
We will then record your contribution in the Wiki:
https://github.com/edrlab/thorium-reader/wiki/Localization-(l10n)-language-translations

**Reply by soulUA** [reply]
Thanks for the advice
- downloaded the file
Ukrainian - Українська

**Reply by danielweck** [reply]
Hello, is the Ukrainian JSON file ready? Are you able to create a Pull Request in this code repository? If not, please just send us the JSON file and we will integrate the locale.

**Comment by soulUA** [comment]
Hello !
The file with the Ukrainian translation is attached to the letter.

--
Ihor

вт, 1 серп. 2023 р. о 23:17 Daniel Weck ***@***.***> пише:

**Reply by danielweck** [reply]
I am sorry I do not understand, what "letter"? Do you mean "email"? (I haven't received anything)

**Comment by soulUA** [comment]
пт, 4 серп. 2023 р. о 10:10 Daniel Weck ***@***.***> пише:

**Comment by Volodymyrivna1** [comment]
Great IDEA !
SLAVA UKRAINI !!!!
